addons/waypoint/functions/fnc_moveVehicleWp.sqf

```diff
@@ -0,0 +1,135 @@
+
+        //making group/unit easier to command
+        _driver setVariable ["lambs_danger_disableAI", true];
+        (group _driver) setVariable ["lambs_danger_disablegroupAI", true];
+
+        _driver enableAI "PATH";
+        _driver domove _pos2;
+        _driver disableAI "AUTOTARGET";
+        _driver disableAI "AUTOCOMBAT";
+        _driver disableAI "TARGET";
+        _driver disableAI "COVER";
+        _driver disableAI "FSM";
+        _driver setBehaviour "AWARE";
+
+        _LeaderVehicle setvariable [QGVAR(DriverWaitTime),(time + 1)];
+        _LeaderVehicle setDriveOnPath (_LeaderVehicle getvariable QGVAR(TargetArrayMovePosition));
+
+        [{
+        //condition
+            params ["_vehicle","_group"];
+            if !(waypointScript [_group,currentWaypoint _group] == QPATHTOF(functions\fnc_moveVehicleWP.sqf)) exitwith {true};
+            if !(alive (driver _vehicle)) exitwith {true};
+            if (lifestate (driver _vehicle) == "INCAPACITATED") exitwith {true};
+            if (_vehicle distance2d (_vehicle getvariable (QGVAR(TargetArrayMovePosition)) select 1) < 8 ) exitwith {true};
+            //refresh AI from "stuck" or move to updated position
+
+            if (_vehicle getvariable QGVAR(DriverWaitTime) < time) then {
+                    _vehicle setvariable [QGVAR(DriverWaitTime),(time + 2)];
+                    if (speed _vehicle < 4) then {
+                        (driver _vehicle) disableAI "PATH";
+                        (driver _vehicle) enableAI "PATH";
+                    };
+                    if !(isFormationLeader (driver _vehicle)) then {
+                        (driver _vehicle) doFollow leader (driver _vehicle);
+                        dostop (driver _vehicle);
+                };
+                (driver _vehicle) domove ( _vehicle getvariable QGVAR(TargetArrayMovePosition) select 1);
+                _vehicle setDriveOnPath ( _vehicle getvariable QGVAR(TargetArrayMovePosition));
+                false;
+            }
+            },{
+        //code when true
+            params ["_vehicle"];
+
+            //fixing driver behaviour where he can ignore all move orders if vehicle don't achive setDriveOnPath destinations before doFollow.
+            private _point1 = _vehicle getrelpos [2,0];
+            private _point2 = _vehicle getrelpos [3,0];
+            _vehicle setDriveOnPath [_point1,_point2];
+
+            [{
+                params ["_vehicle"];
+                (driver _vehicle) doFollow (leader (driver _vehicle));
+                (driver _vehicle) domove (getposATL _vehicle);
+                (driver _vehicle) enableAI "all";
+                (driver _vehicle) setVariable ["lambs_danger_disableAI", false];
+                (group (driver _vehicle)) setVariable ["lambs_danger_disablegroupAI", false];
+                _vehicle setvariable [QGVAR(ActiveVehicleMove),false];
+            },[_vehicle],0.2] call cba_fnc_waitAndExecute;
```

Comment:
Please keep a space after `,`. Also multiple occurrences.
```suggestion
                _vehicle setvariable [QGVAR(ActiveVehicleMove), false];
            }, [_vehicle], 0.2] call cba_fnc_waitAndExecute;
```

Reply:
I see veteran spared his time writing such comments and will have you fix all the issues that will be automatically reported by updated HEMTT. 🤣